datafusion/functions-nested/src/string.rs

```diff
@@ -88,6 +90,7 @@ macro_rules! call_array_function {
     ($DATATYPE:expr, $INCLUDE_LIST:expr) => {{
         match $DATATYPE {
             DataType::List(_) => array_function!(ListArray),
+            DataType::Utf8View => array_function!(StringViewArray),
```

Comment:
nit: maybe put it _after_ DataType::Utf8 match branch

Reply:
Curious why that would matter?

Reply:
if code is consistently ordered, it's easier to follow
it's obviously a nit, ok to ignore